apiary.apib

```diff
@@ -0,0 +1,515 @@
+                        "supplierID": 1,
+                        "supplier": [
+                        {
+                        "name": "Metal Supply Co",
+                        "ABN": "02 999 111 888",
+                        "Logo": "Legoo",
+                        "CreatedAt": "2021-04-25T11:55:34Z",
+                        "UpdatedAt": "2021-04-25T11:55:34Z",
+                        "DeletedAt": "2021-04-25T11:55:34Z"
+                        }]},
+                     {
+                        "supplierID": 2,
+                        "supplier": [
+                        {
+                        "name": "COGS",
+                        "ABN": "98 669 677 128",
+                        "Logo": "Gear",
+                        "CreatedAt": "2021-04-25T11:55:34Z",
+                        "UpdatedAt": "2021-04-25T11:55:34Z",
+                        "DeletedAt": "2021-04-25T11:55:34Z"
+                        }]},
+                    {
+                        "supplierID": 3,
+                        "supplier": [
+                        {
+                        "name": "Tools 4 days",
+                        "ABN": "02 999 111 888",
+                        "Logo": "Tool",
+                        "CreatedAt": "2021-04-25T11:55:34Z",
+                        "UpdatedAt": "2021-04-25T11:55:34Z",
+                        "DeletedAt": "2021-04-25T11:55:34Z"
+                        }]}
+                    {
+                        "supplierID": 4,
+                        "supplier": [
+                        {
+                        "name": "Fall ball",
+                        "ABN": "12 929 121 828",
+                        "Logo": "Ball",
+                        "CreatedAt": "2021-04-25T11:55:34Z",
+                        "UpdatedAt": "2021-04-25T11:55:34Z",
+                        "DeletedAt": "2021-04-25T11:55:34Z"
+                        }]
+                    }
+                ]
+
+### Create a New Supplier [POST]
+
+
++ Request (application/json)
+
+        {
+            "question": "New supplier",
+            "supplier": [
+                "name",
+                "ABN",
+                "Logo"
+            ]
+        }
+
++ Response 201 (application/json)
+
+    + Headers
+ 
+            Location: /supplier/2
+
+    + Body
+
+            {
+                "question": "New supplier",
+                "published_at": "2015-08-05T08:40:51.620Z",
+                "choices": [
+                    {
+                        "supplier": "Swift",
+                        "ABN": 12 222 222 333,
+                        "Logo": "PHHSK"
+                    }, {
+                        "supplier": "Ssact",
+                        "ABN": 12 999 111 333,
+                        "Logo": "PHHSK"
+                    }
+                ]
+            }
+
+ 
+ 
+## Tenders [/tenders]
+
+### Current Tenders [GET]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "tenderID": 1,
```

Comment:
sometimes you are typing like camelCase and sometimes as PascalCase. Adopt one single standard, I would recommend the PascalCase since I already created the database tables as PascalCase